dco-program-faq.md

```diff
@@ -0,0 +1,43 @@
+{{{
+"title": "DCO Program Communication: FAQs",
+"date": "08-01-2019",
+"author": "Lyza Latham",
+"attachments": [],
+"related-products" : [],
+"contentIsHTML": false,
+"sticky": true
+}}}
+
+### Communication: FAQs
+
+<strong>Q: Are managed services still available to my colocation space?</strong>
```

Comment:
You should use markdown `**` instead of strong html tag throughout

Reply:
this has been fixed. thanks.